src/main/java/com/zaxxer/hikari/pool/HikariPool.java

```diff
@@ -365,7 +364,7 @@ public void run()
             while (poolState == POOL_NORMAL && totalConnections.get() < maxPoolSize && getIdleConnections() <= minimumIdle && !addConnection()) {
                // If we got into the loop, addConnection() failed, so we sleep and retry
                quietlySleep(sleepBackoff);
-               sleepBackoff = Math.min(connectionTimeout / 2, (long) ((double) sleepBackoff * 1.5));
+               sleepBackoff = Math.min((long) (connectionTimeout / 2), (long) (sleepBackoff * 1.5));
```

Comment:
it is always a long because ``connectionTimeout`` is long, as defined by Java language specification.